Godeps/Godeps.json

```diff
@@ -6,12 +6,17 @@
 	],
 	"Deps": [
 		{
+			"ImportPath": "code.google.com/p/rog-go/parallel",
```

Comment:
Maybe this import path should be updated to [`github.com/rogpeppe/rog-go/parallel`](https://godoc.org/github.com/rogpeppe/rog-go/parallel) now since code.google.com is deprecated? It's okay if it's out of scope of this PR.